go.mod

```diff
@@ -122,9 +123,11 @@ require (
 	k8s.io/component-base v0.25.1 // indirect
 	k8s.io/kube-openapi v0.0.0-20220803162953-67bda5d908f1 // indirect
 	k8s.io/kubectl v0.25.1 // indirect
-	k8s.io/utils v0.0.0-20220823124924-e9cbc92d1a73 // indirect
 	sigs.k8s.io/json v0.0.0-20220713155537-f223a00ba0e2 // indirect
 	sigs.k8s.io/kustomize/api v0.12.1 // indirect
 	sigs.k8s.io/kustomize/kyaml v0.13.9 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 )
+
+// TODO(cjschaef): Remove before merging
+replace github.com/openshift/api => github.com/cjschaef/api v0.0.0-20221117200200-a44779da707f
```

Comment:
This can be fixed now right?

Reply:
Yep, I'll be working on updating MAPI and MAO with the proper API go mod and perform another round of testing to confirm

Reply:
Tested rebase with custom image. PR has been updated using latest API module.